common/views/components/NewsletterPromo/NewsletterPromo2.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import Button from '../Buttons/Button/Button';
+import { grid, font, classNames } from '../../../utils/classnames';
+import RepeatingLs from '../RepeatingLs/RepeatingLs';
+import Space from '../styled/Space';
+
+const NewsletterPromo = () => (
+  <div className="row">
```

Comment:
Should this pattern go into a component somewhere?

Reply:
You're right, we have the `Layout` component which would be an appropriate fit here.

Reply:
…having said that, each `Layout` component has it's own `container` classed div, and that doesn't work here, so I don't think this can be usefully abstracted further.